### ISSUE TITLE
Using luma.gl buffer as data attributes

### ISSUE THREAD

**Issue by Darune** [uber/deck.gl#4114]
Seeing binary data support in v8 described here: 
https://deck.gl/#/documentation/developer-guide/performance-optimization?section=supply-attributes-directly

I'm trying to use luma.gl Transform API do to some preprocess on my features.
As I understand it, creating new Buffer is not optimal, so I am trying to use the Transform output buffer directly as described in the documentation :

```
new PointCloudLayer({
    data: {
      length : data.pointCount,
      attributes: {
        getPosition: {buffer, size: 3, offset: 0, stride: 24},
(...)
``` 
But supplying attributes this way gives me the following error: 
`Error: invalid getPosition`

This assert seems to be raised from 
`src/lib/attribute/attribute.js:249`
```
(...)
assert(ArrayBuffer.isView(buffer.value), `invalid ${settings.accessor}`);
(...)
```

in my case buffer.value is not defined, so i'm not really sure what i'm doing wrong here.

I simplified my use case to a minimal .html joined here.
[buffer.txt](https://github.com/uber/deck.gl/files/4033993/buffer.txt)

Am I using the feature correctly ? 

changing the `getPosition` attribute to a 
`{value: new Float32Array([2.3522, 48.8566, 10]), size: 3, offset: 0, stride: 12}`
seems to fix the issue, but in my case where I want to use the luma.gl Transform api, 
does this mean that the operation will copy the transform output twice `transform <-> my code <-> deck.gl's own luma buffer` ?

**Comment by 1chandu** [comment]
CC: @Pessimistress 

You should be able to set `Transform` output as external buffer to a Layer attribute.

[Heatmap layer](https://github.com/uber/deck.gl/blob/5c8a156671a780c11ed33831381ad5f9a1b22ca7/modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js#L182) uses `attributes: {attributeName: buffer, ...}`  can you try this way.

**Comment by Pessimistress** [comment]
@1chandu We are moving away from the `[attributeName]: buffer` convention. This seems to be a bug in the Attribute class.

**Comment by Pessimistress** [comment]
Fixed in 8.0.5.

**Comment by Darune** [comment]
Just tried it, thank you for the quick fix / release !
